frontend/src/metabase/query_builder/components/DataSelector/saved-question-picker/SavedQuestionList.tsx

```diff
@@ -29,7 +29,7 @@ const SavedQuestionList = ({
 }: SavedQuestionListProps): JSX.Element => {
   const emptyState = (
     <SavedQuestionListEmptyState>
-      <EmptyState message={t`Nothing here`} icon="all" />
+      <EmptyState message={t`Nothing here`} />
```

Comment:
Now that EmptyState is a typescript file, it won't tolerate this invalid icon name